src/Pages/Wallets.razor

```diff
@@ -284,8 +283,14 @@
 
     private async Task GetData()
     {
-
-        _wallets = await WalletRepository.GetAll();
+        if (ClaimsPrincipal != null && ClaimsPrincipal.IsInRole(ApplicationUserRole.FinanceManager.ToString()))
+        {
+            _wallets = await WalletRepository.GetAll();
+        }
+        else
+        {
+           _wallets = await WalletRepository.GetAvailableWallets(); 
```

Comment:
What's the reason behind this if else?

Reply:
For node managers and admins to not see pending wallets. I think is pointless.